project/plugins.sbt

```diff
@@ -20,4 +20,6 @@ addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.11.0")
 
 // addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.6.1")
 // addSbtPlugin("com.github.cb372" % "sbt-explicit-dependencies" % "0.2.16")
-// addDependencyTreePlugin
+
+// please don't remove or merge uncommented to main
```

Comment:
What do you mean by "don't merge uncommented to main"? Do you mean that the following line should never be commented out (as it was before)?

Reply:
yes, and I broke the rule ;-) will need to fix that first. This pulls in a bunch of libraries, which can slow down the build.